src/components/PackageDetails/index.tsx

```diff
@@ -18,9 +18,7 @@ interface PackageDetailsProps {
  * the Getting Started, Operator Area, and Publisher Area sections
  */
 export function PackageDetails({ assembly, metadata }: PackageDetailsProps) {
-  const isLoading = assembly.loading || metadata.loading;
-
-  if (isLoading || !metadata.data) {
+  if (metadata.loading || !metadata.data) {
```

Comment:
does `assembly.loading` need to be part of this conditional statement?

Reply:
This was before the change Eli made for the dedicated backend, so yeah this needs to go back. (this was how I had improved load states)